src/intermediate.rs

```diff
@@ -52,22 +52,6 @@ impl<'a> IntermediateTypes<'a> {
         }
     }
 
-    pub fn has_ident(&self, ident: &RustIdent) -> bool {
-        let foo: Vec<RustIdent> = self.type_aliases.keys().fold(vec![], |mut acc, alias| {
-            match alias {
-                AliasIdent::Reserved(_) => {},
-                AliasIdent::Rust(ident) => { acc.push(ident.clone()) }
-            };
-            acc
-        });
-        println!("{:?}", self.plain_groups.keys().chain(foo.iter()).chain(self.rust_structs.keys()).chain(self.generic_defs.keys()).chain(self.generic_instances.keys()));
-        self.plain_groups.contains_key(ident)
-         || self.type_aliases.contains_key(&AliasIdent::Rust(ident.clone()))
-         || self.rust_structs.contains_key(ident)
-         || self.generic_defs.contains_key(ident)
-         || self.generic_instances.contains_key(ident)
-    }
```

Comment:
FYI the code removed in this file is all unused. It comes from a previous PR that I forgot to delete and it didn't work because has_ident isn't stable across passes of the parsing.rs